core/src/main/java/life/qbic/identityaccess/application/user/Registration.java

```diff
@@ -67,6 +67,7 @@ public void register(String fullName, String email, char[] rawPassword) {
               response -> registerUserOutput.onUnexpectedFailure(build(response)));
     } catch (Exception e) {
       log.error("User registration failed", e.getCause());
+      log.error(e.getMessage());
```

Comment:
Why not 
```suggestion
      log.error("User registration failed", e);
```

Reply:
Tbh, I dont really understand how the exception logging works. I just observed, that passing the throwable cause does not print anything in the logs, which made it impossible for debugging.